core/src/test/resources/xform_tests/itext_function.xml

```diff
@@ -0,0 +1,48 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml"
+        xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+        xmlns:jr="http://openrosa.org/javarosa">
+    <h:head>
+        <h:title>Constraints for simple and select answers</h:title>
```

Comment:
Might be worth updating, sometimes it is hard to get context from test forms and the title helps